editor/src/components/Generic/ListOptions.vue

```diff
@@ -141,12 +141,19 @@ const ListOptions = defineComponent({
         items.value = clone;
       },
       getData() {
-        return _.map(_.clone(items.value), (i: any) => {
-          return {
-            [this.value]: i.value,
-            [this.label]: i.label,
-          };
-        });
+        if (
+          items.value.length >= 0 &&
+          items.value[0].value != '' &&
```

Comment:
To comparations use  the one of next way === or !===

Reply:
Update code with observations